doc/EFFECTS_JSON.md

```diff
@@ -212,9 +212,16 @@ Because the "dur_add_perc" = 150 in the example above, the second addition adds
 a total value of 250 ticks from the two. This can also be below 100%, and should be able to even be negative, leading to
 future applications decreasing the overall time left.
 
+### Duration control
```

Comment:
If the primary goal is to increase intensity over time, shouldn't the
documentation reflect that?